File: web3rpc/rpc-specs/code-samples/python/eth/others/getProof.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

account = "0xe5cB067E90D5Cd1F8052B83562Ae670bA4A211a8"
keys = ["0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"]
blockNumber = "latest"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_proof(account, keys, blockNumber)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/others/getProof.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const account = "0xe5cB067E90D5Cd1F8052B83562Ae670bA4A211a8";
  const keys = [
    "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
  ];
  const blockNumber = "latest";
  provider.eth
    .getProof(account, keys, blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
